bootstrapvz/providers/azure/tasks/packages.py

```diff
@@ -47,3 +47,9 @@ def run(cls, info):
 			if os.path.isfile(info.manifest.provider['waagent']['conf']):
 				log_check_call(['cp', info.manifest.provider['waagent']['conf'],
 				                os.path.join(info.root, 'etc/waagent.conf')])
+
+		# The Azure Linux agent uses 'useradd' to add users, but SHELL
+		# is set to /bin/sh by default. Set this to /bin/bash instead.
+		from bootstrapvz.common.tools import sed_i
+		useradd_config = os.path.join(info.root, 'etc/default/useradd')
+		sed_i(useradd_config, r'^(SHELL=.*)', r'SHELL=/bin/bash')
```

Comment:
Ah ok. Makes sense. Eventually the agent should be fixed, but bootstrap-vz can help in cases like these to bridge the gaps.